npm/vue/package.json

```diff
@@ -11,7 +11,7 @@
     "cy:run": "node ../../scripts/cypress.js run --component --project ${PWD}",
     "lint": "eslint --ext .js,.jsx,.ts,.tsx,.json,.vue .",
     "test": "yarn cy:run",
-    "tsd": "yarn build && yarn tsc -p test-tsd/tsconfig.tsd.json",
+    "tsd": "yarn build && yarn tsc -p test-tsd/tsconfig.json",
```

Comment:
Renaming `tsconfig.json` makes sure code editors and `tsc` pick up the same config. Otherwise `tsc` would pass, one would still see the error in the editor.